lib/app/app.dart

```diff
@@ -43,7 +44,10 @@ import 'package:nowu/ui/bottom_sheets/explore_filter/explore_filter_sheet.dart';
     CustomRoute(page: FaqView),
     CustomRoute(page: IntroView),
     CustomRoute(page: ProfileSetupView),
-    CustomRoute(page: LoginView),
+    CustomRoute(
+      page: LoginView,
+      transitionsBuilder: TransitionsBuilders.fadeIn,
```

Comment:
Where does this come from? It looks ideal

Reply:
Took it from the docs: https://stacked.filledstacks.com/docs/getting-started/navigation-basics/#custom-transitions

I guess we can unify the transitions in the app and even make all transitions more smooth. If that will be the same animation for all screens I guess it will be a one-liner